source/linux/Foundational_Components/Kernel/Kernel_Drivers/GPIO.rst

```diff
@@ -48,33 +47,186 @@ and interrupt generation.
+
+- For more info on *gpioinfo*, see its man page: https://libgpiod.readthedocs.io/en/latest/gpioinfo.html.
+
+- To see info for a specific GPIO chip, use the '-c' option and the GPIO chip number.
+
+#. Read info for **gpiochip2** (for this example):
+
+      .. code-block:: console
+
+         $ gpioinfo -c 2
+         gpiochip2 - 24 lines:
+                 line   0:       "GPIO_CPSW2_RST"        input
+                 line   1:       "GPIO_CPSW1_RST"        input
+                 line   2:       "PRU_DETECT"            input
+                 line   3:       "MMC1_SD_EN"            output consumer="regulator-3"
+                 line   4:       "VPP_LDO_EN"            input
+                 line   5:       "EXP_PS_3V3_En"         input
+                 line   6:       "EXP_PS_5V0_En"         input
+                 line   7:       "EXP_HAT_DETECT"        input
+                 line   8:       "GPIO_AUD_RSTn"         input
+                 line   9:       "GPIO_eMMC_RSTn"        input
+                 line  10:       "UART1_FET_BUF_EN"      input
+                 line  11:       "WL_LT_EN"              input
+                 line  12:       "GPIO_HDMI_RSTn"        input
+                 line  13:       "CSI_GPIO1"             input
+                 line  14:       "CSI_GPIO2"             input
+                 line  15:       "PRU_3V3_EN"            input
+                 line  16:       "HDMI_INTn"             input consumer="interrupt"
+                 line  17:       "PD_I2C_IRQ"            input
+                 line  18:       "MCASP1_FET_EN"         input
+                 line  19:       "MCASP1_BUF_BT_EN"      input
+                 line  20:       "MCASP1_FET_SEL"        input
+                 line  21:       "UART1_FET_SEL"         input
+                 line  22:       "TSINT#"                input
+                 line  23:       "IO_EXP_TEST_LED"       output
+
+- We see that **IO_EXP_TEST_LED** is connected to *GPIO2_23* on the SK-AM62B-P1 board.
+
+#. Read the value of the GPIO pin:
+
+   .. code-block:: console
+
+         $ gpioget --numeric -c 2 23
+         0
+
+- Use variations of *gpioget* to read the value of the GPIO pin:
+
+   .. code-block:: console
+
+         $ gpioget -c 2 23
+         "23"=inactive
+         $ gpioget IO_EXP_TEST_LED
+         "IO_EXP_TEST_LED"=inactive
+         $ gpioget --numeric IO_EXP_TEST_LED
+         0
+
+- For more info on *gpioget*, see its man page: https://libgpiod.readthedocs.io/en/latest/gpioget.html.
+
+#. Set the GPIO pin to turn on the LED
+
+   .. code-block:: console
+
+         $ gpioset -c 2 23=1
+         ^C
+         $ gpioset -c 2 23=0
+         ^C
+
+- *Note: '^C' (Ctrl + C) interrupts the command, else the command would not terminate on its own. Use '^C' (Ctrl + C) to stop the command, and the new GPIO value will still remain.*
```

Comment:
This note is incorrect.

https://manpages.debian.org/experimental/gpiod/gpioset.1.en.html#NOTE